docs/content/references/yaml-metadata.md

```diff
@@ -0,0 +1,181 @@
+---
+robots: index, follow
+lang: en
+dir: ltr
+breaks: true
+---
```

Comment:
These meta data are not necessary in the docs.

Reply:
Removed